gr-uhd/lib/usrp_sink_impl.cc

```diff
@@ -557,6 +562,43 @@ namespace gr {
     } // end tag_work()
 
     void
+    usrp_sink_impl::_async_loop()
+    {
+      ::uhd::async_metadata_t async_md;
+      pmt::pmt_t dic = pmt::make_dict();
+      // poll the async message queue at a liberal interval – no use wasting CPU
+      while(true)
+      {
+        //have a way to stop this thread
+        boost::this_thread::interruption_point();
+        //receive asynchronous messages
+        if (_tx_stream->recv_async_msg(async_md, 0.25)) {
```

Comment:
Oh, snap. The problem is, we (we == GNU Radio) don't have a sensible UHD minimum version for the 3.7 release cycle, so we kind of have to support it.

We could move this to 3.8, where we want to bump the minimum UHD, or we come up with some smart #ifdef'ing for lower versions.

@marcusmueller, any ideas?